data/tooltips.txt

```diff
@@ -86,8 +86,14 @@ tip "cargo:"
 tip "cargo space:"
 	`How much cargo (commodities, mission cargo, and plundered outfits) this ship can carry.`
 
+tip "permanent cloak:"
```

Comment:
As mentioned in https://github.com/endless-sky/endless-sky/pull/7025#issuecomment-1205425758, let's keep this for a separate PR (since I expect that we better implement this using other keywords).

Reply:
also it could cause issues to have a ship partly cloaked so yeah I'll remove it